modulefiles/ufs_cheyenne.intel

```diff
@@ -11,16 +11,16 @@ module load python/3.7.9
 
 # load programming environment
 module load ncarenv/1.3
-module load intel/2021.2
-module load mpt/2.22
+module load intel/2022.1
+module load mpt/2.25
 module load ncarcompilers/0.5.0
 module unload netcdf
 
-module use /glade/work/epicufsrt/GMTB/tools/hpc-stack-v1.2.0_6eb6/modulefiles/stack
+module use /glade/work/epicufsrt/GMTB/tools/intel/2022.1/hpc-stack-v1.2.0_6eb6/modulefiles/stack
```

Comment:
hpc stack for gnu is located under `hpc-stack-v1.2.0` while intel is under `hpc-stack-v1.2.0_6eb6`. Why `_6eb6` suffix? Is it too late to change this?

Reply:
@DusanJovic-NOAA hpc-stack develop branch has an issue with intel/mpt on Cheyenne. So, I still have to use hash (6eb6) for intel. But gnu has no problem with hpc-stack develop branch. So I have to separate the gnu and intel locations.

Reply:
Ok. Let's hope the issue is resolved soon.

Reply:
@DusanJovic-NOAA Yes, I think we keep moving on ufs-wm side as is. When we try to get away from mpt on cheyenne, the issue can be resolved by itself, I guess.

Reply:
`mpt` doesn't support `mpi_f08`, thus when @DusanJovic-NOAA's PR for that goes in we have to switch anyway